pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -23,12 +23,12 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
-// PipelineResourceResult is used to write key/value pairs to TaskRun pod termination messages.
+// TaskAndTerminationResult is used to write key/value pairs to TaskRun pod termination messages.
 // The key/value pairs may come from the entrypoint binary, or represent a TaskRunResult.
 // If they represent a TaskRunResult, the key is the name of the result and the value is the
 // JSON-serialized value of the result.
-// TODO(#6197): Rename this struct
-type PipelineResourceResult struct {
+// #6434 Renamed from PipelineResourceResult
```

Comment:
no need to link the issue anymore now that this is done. Please add an alias though for backwards compatibility.